lstchain/reco/r0_to_dl1.py

```diff
@@ -838,3 +669,205 @@ def rescale_dl1_charge(event, scaling_factor):
 
     for tel_id, tel in event.dl1.tel.items():
         tel.image *= scaling_factor
+
+
+def event_timing_pointing(dl1_container,
```

Comment:
It might be good to have a test unit for this function, although it is quite complex and long

Reply:
I agree, please be my guest :D

Reply:
:) maybe someday (not too far) but right now I don't have much time. Since we need to change also the tai/utc transformation by using astropy, we could implement the tests there.